pages/5g-core-network-components/tutorials/5gnetwork.md

```diff
@@ -293,10 +308,37 @@ pcap:
   ngap_filename: /tmp/gnb_ngap.pcap   # Path where the NGAP PCAP is stored.
 ```
 
-## Running the 5G Core (Open5GS)
+### Optional: Adding an external GPS reference clock
+
+Ideally the SDR (USRPs) are connected to a 10 MHz external reference clock or GPSDO, although this is not a strict
+requirement. In our tests, many COTS UEs were only able to connect to the gNB without the use of an external reference
```

Comment:
Something not quite right here. 
> In our tests, many COTS UEs were only able to connect to the gNB without the use of an external reference clock.

Either:

> In our tests, many COTS UEs were only able to connect to the gNB **with** the use of an external reference clock.

or:

> In our tests, many COTS UEs were ~~only~~ able to connect to the gNB without the use of an external reference clock.

Not sure which you meant!

Reply:
Thanks, changed to:

> In our tests, many COTS UEs were only able to connect to the gNB with the use of an external reference
clock.